common/rdm/PidStoreHelper.cpp

```diff
@@ -287,5 +289,16 @@ void PidStoreHelper::SupportedPids(
     store->AllPids(descriptors);
   }
 }
+
+
+/**
+ * @brief Compare PIDs by name
+ *
+ * Suitable for use as a sort comparison function
+ */
+bool PidStoreHelper::ComparePidsByName(const PidDescriptor *a,
```

Comment:
There's a tiny bit of me wonders if this should be under PidDescriptor, as you could use it directly via the PidStore for example (rather than the helper), but I'm happy either way.

If you want to add a basic test for it would be great, although this is pretty trivial, so no worries if not. Actually looking further it seems we've got no testing for PidStoreHelper currently!

Reply:
I agree with moving it to the PidDescriptor (it's what's being compared, after all...); I suppose this made more sense when the sorting was happening in the same class.